.github/PULL_REQUEST_TEMPLATE/pull_request.md

```diff
@@ -0,0 +1,26 @@
+Changes proposed in this pull request:
+
+<!--
+Please insert below the list of changes in this pull request. For example:
+
+- Change 1
+- Change 2
+- CHange 3
+-->
+
+-
+
+Related issue:
+
+<!--
+If this pull request is related to any issue, then put its number here please. For example:
+
+- #8
```

Comment:
So this `#` is then converted to nice url automagically @acidicMercury8 ?

![image](https://user-images.githubusercontent.com/3642643/172008005-156704f8-5a59-4ade-be3b-cb0df446942d.png)

Reply:
@PeyTy yes, it was converted automatically